pyvcloud/vcd/org.py

```diff
@@ -410,31 +410,39 @@ def create_user(self,
     def list_roles(self):
         if self.resource is None:
             self.resource = self.client.get_resource(self.href)
-        resource_type = 'adminRole'
+        org_filter = None
+        if self.is_admin:
```

Comment:
list_role and get_role are almost identical in implementation, we should probably clean them up to minimize code duplication.

Reply:
Done.